CONTRIBUTING.rst

```diff
@@ -101,8 +77,7 @@ Before you submit a pull request, check that it meets these guidelines:
 1. The pull request should include tests.
 2. If the pull request adds functionality, the docs should be updated. Put
    your new functionality into a function with a docstring.
-3. The pull request should work for Python 3.7 and 3.8. Check
-   https://travis-ci.com/pyfar/pyfar/pull_requests
+3. Check https://travis-ci.com/pyfar/pyfar/pull_requests
```

Comment:
I think this is a little confusing. I personally would check out travis only, if the checks did not pass (indicated on GitHub below the PR).